build.gradle

```diff
@@ -41,7 +41,7 @@ allprojects {
     // Configure Java to use our chosen language level. Kotlin will automatically pick this up.
     // See https://kotlinlang.org/docs/gradle-configure-project.html#gradle-java-toolchains-support
     plugins.withType(KotlinBasePlugin).configureEach {
-        kotlin.jvmToolchain(8)
+        kotlin.jvmToolchain(11)
```

Comment:
It works but
```
> Task :compileLawnWithQuickstepReleaseKotlin
'compileLawnWithQuickstepReleaseJavaWithJavac' task (current target is 1.8) and 'compileLawnWithQuickstepReleaseKotlin' task (current target is 11) jvm target compatibility should be set to the same Java version.
By default will become an error since Gradle 8.0+! Read more: https://kotl.in/gradle/jvm/target-validation
Consider using JVM toolchain: https://kotl.in/gradle/jvm/toolchain
```